2020-05-18T23:04:04.0405472+07:00  [INF] Entity Framework Core "2.0.1-rtm-125" initialized '"ApplicationDbContext"' using provider '"Microsoft.EntityFrameworkCore.SqlServer"' with options: "MigrationsAssembly=UrencoDB " (9958f5bb)
2020-05-18T23:04:04.2981391+07:00  [FTL] Lỗi khởi tạo cơ sở dữ liệu (3b6074dc)
System.AggregateException: One or more errors occurred. (Could not load file or assembly 'UrencoDB, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.) ---> System.IO.FileNotFoundException: Could not load file or assembly 'UrencoDB, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.
   at System.Reflection.RuntimeAssembly.nLoad(AssemblyName fileName, String codeBase, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, IntPtr ptrLoadContextBinder)
   at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, IntPtr ptrLoadContextBinder)
   at System.Reflection.Assembly.Load(AssemblyName assemblyRef)
   at Microsoft.EntityFrameworkCore.Migrations.Internal.MigrationsAssembly..ctor(ICurrentDbContext currentContext, IDbContextOptions options, IMigrationsIdGenerator idGenerator)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitConstructor(ConstructorCallSite constructorCallSite, ServiceProvider provider)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitScoped(ScopedCallSite scopedCallSite, ServiceProvider provider)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitConstructor(ConstructorCallSite constructorCallSite, ServiceProvider provider)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitScoped(ScopedCallSite scopedCallSite, ServiceProvider provider)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetService[T](IServiceProvider provider)
   at Microsoft.EntityFrameworkCore.RelationalDatabaseFacadeExtensions.GetRelationalService[TService](IInfrastructure`1 databaseFacade)
   at Microsoft.EntityFrameworkCore.RelationalDatabaseFacadeExtensions.MigrateAsync(DatabaseFacade databaseFacade, CancellationToken cancellationToken)
   at DAL.DatabaseInitializer.SeedAsync() in D:\PMQLHienVat\source\QLNS\DAL\DatabaseInitializer.cs:line 46
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at QLNS.Program.Main(String[] args) in D:\PMQLHienVat\source\QLNS\QLNS\Program.cs:line 35
---> (Inner Exception #0) System.IO.FileNotFoundException: Could not load file or assembly 'UrencoDB, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.
File name: 'UrencoDB, Culture=neutral, PublicKeyToken=null'
   at System.Reflection.RuntimeAssembly.nLoad(AssemblyName fileName, String codeBase, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, IntPtr ptrLoadContextBinder)
   at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, IntPtr ptrLoadContextBinder)
   at System.Reflection.Assembly.Load(AssemblyName assemblyRef)
   at Microsoft.EntityFrameworkCore.Migrations.Internal.MigrationsAssembly..ctor(ICurrentDbContext currentContext, IDbContextOptions options, IMigrationsIdGenerator idGenerator)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitConstructor(ConstructorCallSite constructorCallSite, ServiceProvider provider)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitScoped(ScopedCallSite scopedCallSite, ServiceProvider provider)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitConstructor(ConstructorCallSite constructorCallSite, ServiceProvider provider)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitScoped(ScopedCallSite scopedCallSite, ServiceProvider provider)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetService[T](IServiceProvider provider)
   at Microsoft.EntityFrameworkCore.RelationalDatabaseFacadeExtensions.GetRelationalService[TService](IInfrastructure`1 databaseFacade)
   at Microsoft.EntityFrameworkCore.RelationalDatabaseFacadeExtensions.MigrateAsync(DatabaseFacade databaseFacade, CancellationToken cancellationToken)
   at DAL.DatabaseInitializer.SeedAsync() in D:\PMQLHienVat\source\QLNS\DAL\DatabaseInitializer.cs:line 46

<---

2020-05-18T23:04:05.5954181+07:00 0HLVR8KBD6M2M:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:55842/   (ca22a1cb)
2020-05-18T23:04:06.9391654+07:00 0HLVR8KBD6M2M:00000001 [INF] Executing action method "QLNS.Controllers.HomeController.Index (QLNS)" with arguments (null) - ModelState is Valid (ba7f4ac2)
2020-05-18T23:04:20.2643194+07:00 0HLVR8KBD6M2M:00000001 [INF] Executing ViewResult, running view at path "/Views/Home/Index.cshtml". (9707eebe)
2020-05-18T23:04:24.2976963+07:00 0HLVR8KBD6M2M:00000001 [INF] Executed action "QLNS.Controllers.HomeController.Index (QLNS)" in 17455.8744ms (afa2e885)
2020-05-18T23:04:24.3530842+07:00 0HLVR8KBD6M2M:00000001 [INF] Request finished in 18860.271ms 200 text/html; charset=utf-8 (791a596a)
2020-05-18T23:04:24.4509120+07:00 0HLVR8KBD6M2M:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:55842/dist/main-client.js?v=XOtcadF6T_zXZytOTSx3IUDMkMDFUlNlWTcEsaquSPQ   (ca22a1cb)
2020-05-18T23:04:24.4736391+07:00 0HLVR8KBD6M2N:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:55842/dist/vendor.css?v=QOq4-etCsJX8m9f_uO-JvgMS6xz8jLZCOH8wJGYZOR0   (ca22a1cb)
2020-05-18T23:04:24.5325261+07:00 0HLVR8KBD6M2O:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:55842/dist/vendor.js?v=FwYw6gAJD-OCZqk1GTiv1cCp_cSE_F6L35nFyXTyXgM   (ca22a1cb)
2020-05-18T23:04:53.5141796+07:00  [INF] webpack built 1aa7c2bca7a7ec26987f in 52037ms (1e81ee17)
2020-05-18T23:04:54.3308945+07:00 0HLVR8KBD6M2N:00000001 [INF] Sending file. Request path: '"/dist/vendor.css"'. Physical path: '"D:\PMQLHienVat\source\QLNS\QLNS\wwwroot\dist\vendor.css"' (27b0a520)
2020-05-18T23:04:54.3347859+07:00 0HLVR8KBD6M2O:00000001 [INF] Sending file. Request path: '"/dist/vendor.js"'. Physical path: '"D:\PMQLHienVat\source\QLNS\QLNS\wwwroot\dist\vendor.js"' (27b0a520)
2020-05-18T23:04:54.3806385+07:00 0HLVR8KBD6M2N:00000001 [INF] Request finished in 29906.9952ms 200 text/css (791a596a)
2020-05-18T23:04:54.6314434+07:00 0HLVR8KBD6M2O:00000001 [INF] Request finished in 30098.9265ms 200 application/javascript (791a596a)
2020-05-18T23:04:56.1235427+07:00 0HLVR8KBD6M2M:00000002 [INF] Request finished in 31672.7427ms 200 application/javascript; charset=UTF-8 (791a596a)
2020-05-18T23:05:07.6255885+07:00 0HLVR8KBD6M2N:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:55842/images/icon/title-sw.png   (ca22a1cb)
2020-05-18T23:05:07.6257146+07:00 0HLVR8KBD6M2O:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:55842/images/icon/bg.png   (ca22a1cb)
2020-05-18T23:05:07.6257796+07:00 0HLVR8KBD6M2N:00000002 [INF] Sending file. Request path: '"/images/icon/title-sw.png"'. Physical path: '"D:\PMQLHienVat\source\QLNS\QLNS\wwwroot\images\icon\title-sw.png"' (27b0a520)
2020-05-18T23:05:07.6258385+07:00 0HLVR8KBD6M2O:00000002 [INF] Sending file. Request path: '"/images/icon/bg.png"'. Physical path: '"D:\PMQLHienVat\source\QLNS\QLNS\wwwroot\images\icon\bg.png"' (27b0a520)
2020-05-18T23:05:07.6261205+07:00 0HLVR8KBD6M2M:00000003 [INF] Request starting HTTP/1.1 GET http://localhost:55842/images/icon/khung.png   (ca22a1cb)
2020-05-18T23:05:07.6262888+07:00 0HLVR8KBD6M2M:00000003 [INF] Sending file. Request path: '"/images/icon/khung.png"'. Physical path: '"D:\PMQLHienVat\source\QLNS\QLNS\wwwroot\images\icon\khung.png"' (27b0a520)
2020-05-18T23:05:07.6283482+07:00 0HLVR8KBD6M2M:00000003 [INF] Request finished in 2.0866ms 200 image/png (791a596a)
2020-05-18T23:05:07.6412256+07:00 0HLVR8KBD6M2N:00000002 [INF] Request finished in 15.6567ms 200 image/png (791a596a)
2020-05-18T23:05:07.6748532+07:00 0HLVR8KBD6M2M:00000004 [INF] Request starting HTTP/1.1 GET http://localhost:55842/dist/__webpack_hmr   (ca22a1cb)
2020-05-18T23:05:07.6755979+07:00 0HLVR8KBD6M2N:00000003 [INF] Request starting HTTP/1.1 GET http://localhost:55842/images/icon/mitec-hrm.png   (ca22a1cb)
2020-05-18T23:05:07.6757637+07:00 0HLVR8KBD6M2N:00000003 [INF] Sending file. Request path: '"/images/icon/mitec-hrm.png"'. Physical path: '"D:\PMQLHienVat\source\QLNS\QLNS\wwwroot\images\icon\mitec-hrm.png"' (27b0a520)
2020-05-18T23:05:07.6918240+07:00 0HLVR8KBD6M2P:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:55842/images/icon/logo.png   (ca22a1cb)
2020-05-18T23:05:07.6921272+07:00 0HLVR8KBD6M2P:00000001 [INF] Sending file. Request path: '"/images/icon/logo.png"'. Physical path: '"D:\PMQLHienVat\source\QLNS\QLNS\wwwroot\images\icon\logo.png"' (27b0a520)
2020-05-18T23:05:07.6927255+07:00 0HLVR8KBD6M2N:00000003 [INF] Request finished in 17.1654ms 200 image/png (791a596a)
2020-05-18T23:05:07.7108879+07:00 0HLVR8KBD6M2P:00000001 [INF] Request finished in 19.0935ms 200 image/png (791a596a)
2020-05-18T23:05:07.8267278+07:00 0HLVR8KBD6M2O:00000002 [INF] Request finished in 201.0057ms 200 image/png (791a596a)
2020-05-18T23:05:07.8422090+07:00 0HLVR8KBD6M2N:00000004 [INF] Request starting HTTP/1.1 GET http://localhost:55842/images/font/utm-avo.woff   (ca22a1cb)
2020-05-18T23:05:07.8429432+07:00 0HLVR8KBD6M2P:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:55842/images/font/utm-avobold.woff   (ca22a1cb)
2020-05-18T23:05:07.8536792+07:00 0HLVR8KBD6M2N:00000004 [INF] Policy execution successful. (0204925d)
2020-05-18T23:05:07.8538097+07:00 0HLVR8KBD6M2P:00000002 [INF] Policy execution successful. (0204925d)
2020-05-18T23:05:07.8594850+07:00 0HLVR8KBD6M2P:00000002 [INF] Sending file. Request path: '"/images/font/utm-avobold.woff"'. Physical path: '"D:\PMQLHienVat\source\QLNS\QLNS\wwwroot\images\font\utm-avobold.woff"' (27b0a520)
2020-05-18T23:05:07.8594850+07:00 0HLVR8KBD6M2N:00000004 [INF] Sending file. Request path: '"/images/font/utm-avo.woff"'. Physical path: '"D:\PMQLHienVat\source\QLNS\QLNS\wwwroot\images\font\utm-avo.woff"' (27b0a520)
2020-05-18T23:05:07.9392535+07:00 0HLVR8KBD6M2N:00000004 [INF] Request finished in 97.0544ms 200 application/font-woff (791a596a)
2020-05-18T23:05:07.9432471+07:00 0HLVR8KBD6M2P:00000002 [INF] Request finished in 100.274ms 200 application/font-woff (791a596a)
2020-05-18T23:05:08.2374070+07:00 0HLVR8KBD6M2P:00000003 [INF] Request starting HTTP/1.1 GET http://localhost:55842/favicon.ico   (ca22a1cb)
2020-05-18T23:05:08.2375975+07:00 0HLVR8KBD6M2P:00000003 [INF] Sending file. Request path: '"/favicon.ico"'. Physical path: '"D:\PMQLHienVat\source\QLNS\QLNS\wwwroot\favicon.ico"' (27b0a520)
2020-05-18T23:05:08.2544068+07:00 0HLVR8KBD6M2P:00000003 [INF] Request finished in 17.0183ms 200 image/x-icon (791a596a)
2020-05-18T23:05:41.3443024+07:00 0HLVR8KBD6M2N:00000005 [INF] Request starting HTTP/1.1 POST http://localhost:55842/connect/token application/x-www-form-urlencoded 155 (ca22a1cb)
2020-05-18T23:05:41.3446897+07:00 0HLVR8KBD6M2N:00000005 [INF] Policy execution successful. (0204925d)
2020-05-18T23:05:41.4883698+07:00 0HLVR8KBD6M2N:00000005 [INF] The token request was successfully extracted from the HTTP request: "{
  \"username\": \"trungvu18\",
  \"password\": \"[removed for security reasons]\",
  \"grant_type\": \"password\",
  \"scope\": \"openid email phone profile offline_access roles\",
  \"resource\": \"http://localhost:55842\"
}". (ebc01284)
2020-05-18T23:05:41.5023589+07:00 0HLVR8KBD6M2N:00000005 [INF] The token request was successfully validated. (d7e51e65)
2020-05-18T23:05:41.6179881+07:00 0HLVR8KBD6M2N:00000005 [INF] Executing action method "QLNS.Controllers.AuthorizationController.Exchange (QLNS)" with arguments (["{
  \"username\": \"trungvu18\",
  \"password\": \"[removed for security reasons]\",
  \"grant_type\": \"password\",
  \"scope\": \"openid email phone profile offline_access roles\",
  \"resource\": \"http://localhost:55842\"
}"]) - ModelState is Valid (ba7f4ac2)
2020-05-18T23:05:41.6279230+07:00 0HLVR8KBD6M2N:00000005 [INF] Entity Framework Core "2.0.1-rtm-125" initialized '"ApplicationDbContext"' using provider '"Microsoft.EntityFrameworkCore.SqlServer"' with options: "MigrationsAssembly=UrencoDB " (9958f5bb)
2020-05-18T23:05:42.3421975+07:00 0HLVR8KBD6M2N:00000005 [ERR] An error occurred using the connection to database '"UrencoDb"' on server '"123.25.21.54"'. (5fc3407c)
System.Data.SqlClient.SqlException (0x80131904): Login failed for user 'urenco'.
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
ClientConnectionId:1cf64eeb-519a-4575-acb9-378aa3ae533e
Error Number:18456,State:1,Class:14
2020-05-18T23:05:42.4104770+07:00 0HLVR8KBD6M2N:00000005 [ERR] An exception occurred in the database while iterating the results of a query for context type '"DAL.ApplicationDbContext"'."
""System.Data.SqlClient.SqlException (0x80131904): Login failed for user 'urenco'.
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.BufferlessMoveNext(Boolean buffer, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.FirstOrDefault_[TSource](IAsyncEnumerable`1 source, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.TaskResultAsyncEnumerable`1.Enumerator.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.SelectEnumerableAsyncIterator`2.MoveNextCore(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.AsyncIterator`1.MoveNext(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.MoveNext(CancellationToken cancellationToken)
ClientConnectionId:1cf64eeb-519a-4575-acb9-378aa3ae533e
Error Number:18456,State:1,Class:14" (64e2593c)
System.Data.SqlClient.SqlException (0x80131904): Login failed for user 'urenco'.
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.BufferlessMoveNext(Boolean buffer, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.FirstOrDefault_[TSource](IAsyncEnumerable`1 source, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.TaskResultAsyncEnumerable`1.Enumerator.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.SelectEnumerableAsyncIterator`2.MoveNextCore(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.AsyncIterator`1.MoveNext(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.MoveNext(CancellationToken cancellationToken)
ClientConnectionId:1cf64eeb-519a-4575-acb9-378aa3ae533e
Error Number:18456,State:1,Class:14
2020-05-18T23:05:42.4468859+07:00 0HLVR8KBD6M2N:00000005 [ERR] An exception occurred in the database while iterating the results of a query for context type '"DAL.ApplicationDbContext"'."
""System.Data.SqlClient.SqlException (0x80131904): Login failed for user 'urenco'.
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.BufferlessMoveNext(Boolean buffer, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.FirstOrDefault_[TSource](IAsyncEnumerable`1 source, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.TaskResultAsyncEnumerable`1.Enumerator.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.SelectEnumerableAsyncIterator`2.MoveNextCore(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.AsyncIterator`1.MoveNext(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.MoveNext(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryCompiler.ExecuteSingletonAsyncQuery[TResult](QueryContext queryContext, Func`2 compiledQuery, IDiagnosticsLogger`1 logger, Type contextType)
ClientConnectionId:1cf64eeb-519a-4575-acb9-378aa3ae533e
Error Number:18456,State:1,Class:14" (64e2593c)
System.Data.SqlClient.SqlException (0x80131904): Login failed for user 'urenco'.
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.BufferlessMoveNext(Boolean buffer, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.FirstOrDefault_[TSource](IAsyncEnumerable`1 source, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.TaskResultAsyncEnumerable`1.Enumerator.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.SelectEnumerableAsyncIterator`2.MoveNextCore(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.AsyncIterator`1.MoveNext(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.MoveNext(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryCompiler.ExecuteSingletonAsyncQuery[TResult](QueryContext queryContext, Func`2 compiledQuery, IDiagnosticsLogger`1 logger, Type contextType)
   at QLNS.Controllers.AuthorizationController.Exchange(OpenIdConnectRequest request) in D:\PMQLHienVat\source\QLNS\QLNS\Controllers\AuthorizationController.cs:line 54
ClientConnectionId:1cf64eeb-519a-4575-acb9-378aa3ae533e
Error Number:18456,State:1,Class:14
2020-05-18T23:05:42.4614318+07:00 0HLVR8KBD6M2N:00000005 [INF] Executed action "QLNS.Controllers.AuthorizationController.Exchange (QLNS)" in 895.689ms (afa2e885)
2020-05-18T23:05:42.4908232+07:00 0HLVR8KBD6M2N:00000005 [ERR] An unhandled exception has occurred: Login failed for user 'urenco'. (9b264e06)
System.Data.SqlClient.SqlException (0x80131904): Login failed for user 'urenco'.
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.BufferlessMoveNext(Boolean buffer, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.FirstOrDefault_[TSource](IAsyncEnumerable`1 source, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.TaskResultAsyncEnumerable`1.Enumerator.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.SelectEnumerableAsyncIterator`2.MoveNextCore(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.AsyncIterator`1.MoveNext(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.MoveNext(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryCompiler.ExecuteSingletonAsyncQuery[TResult](QueryContext queryContext, Func`2 compiledQuery, IDiagnosticsLogger`1 logger, Type contextType)
   at QLNS.Controllers.AuthorizationController.Exchange(OpenIdConnectRequest request) in D:\PMQLHienVat\source\QLNS\QLNS\Controllers\AuthorizationController.cs:line 54
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIIndexMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.Invoke(HttpContext context)
ClientConnectionId:1cf64eeb-519a-4575-acb9-378aa3ae533e
Error Number:18456,State:1,Class:14
2020-05-18T23:05:42.5169240+07:00 0HLVR8KBD6M2N:00000005 [INF] Request finished in 1172.6471ms 500 application/json (791a596a)
2020-05-18T23:05:49.8082594+07:00 0HLVR8KBD6M2O:00000003 [INF] Request starting HTTP/1.1 POST http://localhost:55842/connect/token application/x-www-form-urlencoded 155 (ca22a1cb)
2020-05-18T23:05:49.8085348+07:00 0HLVR8KBD6M2O:00000003 [INF] Policy execution successful. (0204925d)
2020-05-18T23:05:49.8097061+07:00 0HLVR8KBD6M2O:00000003 [INF] The token request was successfully extracted from the HTTP request: "{
  \"username\": \"trungvu18\",
  \"password\": \"[removed for security reasons]\",
  \"grant_type\": \"password\",
  \"scope\": \"openid email phone profile offline_access roles\",
  \"resource\": \"http://localhost:55842\"
}". (ebc01284)
2020-05-18T23:05:49.8100220+07:00 0HLVR8KBD6M2O:00000003 [INF] The token request was successfully validated. (d7e51e65)
2020-05-18T23:05:49.8155899+07:00 0HLVR8KBD6M2O:00000003 [INF] Executing action method "QLNS.Controllers.AuthorizationController.Exchange (QLNS)" with arguments (["{
  \"username\": \"trungvu18\",
  \"password\": \"[removed for security reasons]\",
  \"grant_type\": \"password\",
  \"scope\": \"openid email phone profile offline_access roles\",
  \"resource\": \"http://localhost:55842\"
}"]) - ModelState is Valid (ba7f4ac2)
2020-05-18T23:05:49.8165634+07:00 0HLVR8KBD6M2O:00000003 [INF] Entity Framework Core "2.0.1-rtm-125" initialized '"ApplicationDbContext"' using provider '"Microsoft.EntityFrameworkCore.SqlServer"' with options: "MigrationsAssembly=UrencoDB " (9958f5bb)
2020-05-18T23:05:49.9021557+07:00 0HLVR8KBD6M2O:00000003 [ERR] An error occurred using the connection to database '"UrencoDb"' on server '"123.25.21.54"'. (5fc3407c)
System.Data.SqlClient.SqlException (0x80131904): Login failed for user 'urenco'.
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.BufferlessMoveNext(Boolean buffer, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
ClientConnectionId:100bdccb-b841-4666-8094-d3619f45d519
Error Number:18456,State:1,Class:14
2020-05-18T23:05:49.9274741+07:00 0HLVR8KBD6M2O:00000003 [ERR] An exception occurred in the database while iterating the results of a query for context type '"DAL.ApplicationDbContext"'."
""System.Data.SqlClient.SqlException (0x80131904): Login failed for user 'urenco'.
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.BufferlessMoveNext(Boolean buffer, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.FirstOrDefault_[TSource](IAsyncEnumerable`1 source, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.TaskResultAsyncEnumerable`1.Enumerator.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.SelectEnumerableAsyncIterator`2.MoveNextCore(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.AsyncIterator`1.MoveNext(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.MoveNext(CancellationToken cancellationToken)
ClientConnectionId:100bdccb-b841-4666-8094-d3619f45d519
Error Number:18456,State:1,Class:14" (64e2593c)
System.Data.SqlClient.SqlException (0x80131904): Login failed for user 'urenco'.
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.BufferlessMoveNext(Boolean buffer, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.FirstOrDefault_[TSource](IAsyncEnumerable`1 source, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.TaskResultAsyncEnumerable`1.Enumerator.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.SelectEnumerableAsyncIterator`2.MoveNextCore(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.AsyncIterator`1.MoveNext(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.MoveNext(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryCompiler.ExecuteSingletonAsyncQuery[TResult](QueryContext queryContext, Func`2 compiledQuery, IDiagnosticsLogger`1 logger, Type contextType)
ClientConnectionId:100bdccb-b841-4666-8094-d3619f45d519
Error Number:18456,State:1,Class:14
2020-05-18T23:05:49.9575515+07:00 0HLVR8KBD6M2O:00000003 [ERR] An exception occurred in the database while iterating the results of a query for context type '"DAL.ApplicationDbContext"'."
""System.Data.SqlClient.SqlException (0x80131904): Login failed for user 'urenco'.
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.BufferlessMoveNext(Boolean buffer, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.FirstOrDefault_[TSource](IAsyncEnumerable`1 source, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.TaskResultAsyncEnumerable`1.Enumerator.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.SelectEnumerableAsyncIterator`2.MoveNextCore(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.AsyncIterator`1.MoveNext(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.MoveNext(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryCompiler.ExecuteSingletonAsyncQuery[TResult](QueryContext queryContext, Func`2 compiledQuery, IDiagnosticsLogger`1 logger, Type contextType)
ClientConnectionId:100bdccb-b841-4666-8094-d3619f45d519
Error Number:18456,State:1,Class:14" (64e2593c)
System.Data.SqlClient.SqlException (0x80131904): Login failed for user 'urenco'.
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.BufferlessMoveNext(Boolean buffer, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.FirstOrDefault_[TSource](IAsyncEnumerable`1 source, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.TaskResultAsyncEnumerable`1.Enumerator.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.SelectEnumerableAsyncIterator`2.MoveNextCore(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.AsyncIterator`1.MoveNext(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.MoveNext(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryCompiler.ExecuteSingletonAsyncQuery[TResult](QueryContext queryContext, Func`2 compiledQuery, IDiagnosticsLogger`1 logger, Type contextType)
   at QLNS.Controllers.AuthorizationController.Exchange(OpenIdConnectRequest request) in D:\PMQLHienVat\source\QLNS\QLNS\Controllers\AuthorizationController.cs:line 54
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIIndexMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.Invoke(HttpContext context)
ClientConnectionId:100bdccb-b841-4666-8094-d3619f45d519
Error Number:18456,State:1,Class:14
2020-05-18T23:05:49.9584165+07:00 0HLVR8KBD6M2O:00000003 [INF] Executed action "QLNS.Controllers.AuthorizationController.Exchange (QLNS)" in 143.5521ms (afa2e885)
2020-05-18T23:05:49.9715911+07:00 0HLVR8KBD6M2O:00000003 [ERR] An unhandled exception has occurred: Login failed for user 'urenco'. (9b264e06)
System.Data.SqlClient.SqlException (0x80131904): Login failed for user 'urenco'.
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.BufferlessMoveNext(Boolean buffer, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.FirstOrDefault_[TSource](IAsyncEnumerable`1 source, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.TaskResultAsyncEnumerable`1.Enumerator.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.SelectEnumerableAsyncIterator`2.MoveNextCore(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.AsyncIterator`1.MoveNext(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.MoveNext(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryCompiler.ExecuteSingletonAsyncQuery[TResult](QueryContext queryContext, Func`2 compiledQuery, IDiagnosticsLogger`1 logger, Type contextType)
   at QLNS.Controllers.AuthorizationController.Exchange(OpenIdConnectRequest request) in D:\PMQLHienVat\source\QLNS\QLNS\Controllers\AuthorizationController.cs:line 54
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIIndexMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.Invoke(HttpContext context)
ClientConnectionId:100bdccb-b841-4666-8094-d3619f45d519
Error Number:18456,State:1,Class:14
2020-05-18T23:05:49.9723242+07:00 0HLVR8KBD6M2O:00000003 [INF] Request finished in 164.168ms 500 application/json (791a596a)
2020-05-18T23:07:36.8714618+07:00  [INF] Entity Framework Core "2.0.1-rtm-125" initialized '"ApplicationDbContext"' using provider '"Microsoft.EntityFrameworkCore.SqlServer"' with options: "MigrationsAssembly=UrencoDB " (9958f5bb)
2020-05-18T23:07:36.9648065+07:00  [FTL] Lỗi khởi tạo cơ sở dữ liệu (3b6074dc)
System.AggregateException: One or more errors occurred. (Could not load file or assembly 'UrencoDB, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.) ---> System.IO.FileNotFoundException: Could not load file or assembly 'UrencoDB, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.
   at System.Reflection.RuntimeAssembly.nLoad(AssemblyName fileName, String codeBase, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, IntPtr ptrLoadContextBinder)
   at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, IntPtr ptrLoadContextBinder)
   at System.Reflection.Assembly.Load(AssemblyName assemblyRef)
   at Microsoft.EntityFrameworkCore.Migrations.Internal.MigrationsAssembly..ctor(ICurrentDbContext currentContext, IDbContextOptions options, IMigrationsIdGenerator idGenerator)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitConstructor(ConstructorCallSite constructorCallSite, ServiceProvider provider)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitScoped(ScopedCallSite scopedCallSite, ServiceProvider provider)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitConstructor(ConstructorCallSite constructorCallSite, ServiceProvider provider)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitScoped(ScopedCallSite scopedCallSite, ServiceProvider provider)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetService[T](IServiceProvider provider)
   at Microsoft.EntityFrameworkCore.RelationalDatabaseFacadeExtensions.GetRelationalService[TService](IInfrastructure`1 databaseFacade)
   at Microsoft.EntityFrameworkCore.RelationalDatabaseFacadeExtensions.MigrateAsync(DatabaseFacade databaseFacade, CancellationToken cancellationToken)
   at DAL.DatabaseInitializer.SeedAsync() in D:\PMQLHienVat\source\QLNS\DAL\DatabaseInitializer.cs:line 46
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at QLNS.Program.Main(String[] args) in D:\PMQLHienVat\source\QLNS\QLNS\Program.cs:line 35
---> (Inner Exception #0) System.IO.FileNotFoundException: Could not load file or assembly 'UrencoDB, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.
File name: 'UrencoDB, Culture=neutral, PublicKeyToken=null'
   at System.Reflection.RuntimeAssembly.nLoad(AssemblyName fileName, String codeBase, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, IntPtr ptrLoadContextBinder)
   at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, IntPtr ptrLoadContextBinder)
   at System.Reflection.Assembly.Load(AssemblyName assemblyRef)
   at Microsoft.EntityFrameworkCore.Migrations.Internal.MigrationsAssembly..ctor(ICurrentDbContext currentContext, IDbContextOptions options, IMigrationsIdGenerator idGenerator)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitConstructor(ConstructorCallSite constructorCallSite, ServiceProvider provider)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitScoped(ScopedCallSite scopedCallSite, ServiceProvider provider)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitConstructor(ConstructorCallSite constructorCallSite, ServiceProvider provider)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitScoped(ScopedCallSite scopedCallSite, ServiceProvider provider)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetService[T](IServiceProvider provider)
   at Microsoft.EntityFrameworkCore.RelationalDatabaseFacadeExtensions.GetRelationalService[TService](IInfrastructure`1 databaseFacade)
   at Microsoft.EntityFrameworkCore.RelationalDatabaseFacadeExtensions.MigrateAsync(DatabaseFacade databaseFacade, CancellationToken cancellationToken)
   at DAL.DatabaseInitializer.SeedAsync() in D:\PMQLHienVat\source\QLNS\DAL\DatabaseInitializer.cs:line 46

<---

2020-05-18T23:07:37.4825980+07:00 0HLVR8MAKOAMM:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:55842/login   (ca22a1cb)
2020-05-18T23:07:38.2566401+07:00 0HLVR8MAKOAMM:00000001 [INF] Executing action method "QLNS.Controllers.HomeController.Index (QLNS)" with arguments (null) - ModelState is Valid (ba7f4ac2)
2020-05-18T23:07:45.6303607+07:00 0HLVR8MAKOAMN:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:55842/dist/__webpack_hmr   (ca22a1cb)
2020-05-18T23:07:45.8707311+07:00 0HLVR8MAKOAMM:00000001 [INF] Executing ViewResult, running view at path "/Views/Home/Index.cshtml". (9707eebe)
2020-05-18T23:07:46.9201392+07:00 0HLVR8MAKOAMM:00000001 [INF] Executed action "QLNS.Controllers.HomeController.Index (QLNS)" in 8718.8906ms (afa2e885)
2020-05-18T23:07:46.9601831+07:00 0HLVR8MAKOAMO:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:55842/dist/main-client.js?v=GJvw4Xd44QumXaKgyzdeiOOge5bJP624l1Mje6XDNd0   (ca22a1cb)
2020-05-18T23:07:46.9620695+07:00 0HLVR8MAKOAMM:00000001 [INF] Request finished in 9526.4667ms 200 text/html; charset=utf-8 (791a596a)
2020-05-18T23:08:15.8119139+07:00  [INF] webpack built 1aa7c2bca7a7ec26987f in 40390ms (f6e0a5d8)
2020-05-18T23:08:16.3263198+07:00 0HLVR8MAKOAMN:00000001 [INF] Request finished in 30696.0206ms 200 text/event-stream; charset=utf-8 (791a596a)
2020-05-18T23:08:16.7288343+07:00 0HLVR8MAKOAMO:00000001 [INF] Request finished in 29768.5646ms 200 application/javascript; charset=UTF-8 (791a596a)
2020-05-18T23:08:23.4276737+07:00 0HLVR8MAKOAMM:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:55842/dist/__webpack_hmr   (ca22a1cb)
2020-05-18T23:08:23.5453475+07:00 0HLVR8MAKOAMO:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:55842/favicon.ico   (ca22a1cb)
2020-05-18T23:08:23.5795458+07:00 0HLVR8MAKOAMO:00000002 [INF] Sending file. Request path: '"/favicon.ico"'. Physical path: '"D:\PMQLHienVat\source\QLNS\QLNS\wwwroot\favicon.ico"' (27b0a520)
2020-05-18T23:08:23.6072421+07:00 0HLVR8MAKOAMO:00000002 [INF] Request finished in 61.9275ms 200 image/x-icon (791a596a)
2020-05-18T23:08:35.8160841+07:00 0HLVR8MAKOAMO:00000003 [INF] Request starting HTTP/1.1 POST http://localhost:55842/connect/token application/x-www-form-urlencoded 155 (ca22a1cb)
2020-05-18T23:08:35.8258060+07:00 0HLVR8MAKOAMO:00000003 [INF] Policy execution successful. (0204925d)
2020-05-18T23:08:35.9688803+07:00 0HLVR8MAKOAMO:00000003 [INF] The token request was successfully extracted from the HTTP request: "{
  \"username\": \"trungvu18\",
  \"password\": \"[removed for security reasons]\",
  \"grant_type\": \"password\",
  \"scope\": \"openid email phone profile offline_access roles\",
  \"resource\": \"http://localhost:55842\"
}". (ebc01284)
2020-05-18T23:08:35.9845057+07:00 0HLVR8MAKOAMO:00000003 [INF] The token request was successfully validated. (d7e51e65)
2020-05-18T23:08:36.0855733+07:00 0HLVR8MAKOAMO:00000003 [INF] Executing action method "QLNS.Controllers.AuthorizationController.Exchange (QLNS)" with arguments (["{
  \"username\": \"trungvu18\",
  \"password\": \"[removed for security reasons]\",
  \"grant_type\": \"password\",
  \"scope\": \"openid email phone profile offline_access roles\",
  \"resource\": \"http://localhost:55842\"
}"]) - ModelState is Valid (ba7f4ac2)
2020-05-18T23:08:36.0944684+07:00 0HLVR8MAKOAMO:00000003 [INF] Entity Framework Core "2.0.1-rtm-125" initialized '"ApplicationDbContext"' using provider '"Microsoft.EntityFrameworkCore.SqlServer"' with options: "MigrationsAssembly=UrencoDB " (9958f5bb)
2020-05-18T23:08:36.8827148+07:00 0HLVR8MAKOAMO:00000003 [INF] Executed DbCommand ("118"ms) [Parameters=["@__normalizedEmail_0='?' (Size = 256)"], CommandType='Text', CommandTimeout='30']"
""SELECT TOP(1) [u].[Id], [u].[AccessFailedCount], [u].[ConcurrencyStamp], [u].[Configuration], [u].[CreatedBy], [u].[CreatedDate], [u].[Email], [u].[EmailConfirmed], [u].[FullName], [u].[IsEnabled], [u].[JobTitle], [u].[LockoutEnabled], [u].[LockoutEnd], [u].[NormalizedEmail], [u].[NormalizedUserName], [u].[PasswordHash], [u].[PhoneNumber], [u].[PhoneNumberConfirmed], [u].[SecurityStamp], [u].[TwoFactorEnabled], [u].[UpdatedBy], [u].[UpdatedDate], [u].[UserName]
FROM [AspNetUsers] AS [u]
WHERE [u].[NormalizedEmail] = @__normalizedEmail_0" (0723d8ff)
2020-05-18T23:08:36.9618444+07:00 0HLVR8MAKOAMO:00000003 [INF] Executed DbCommand ("7"ms) [Parameters=["@__normalizedUserName_0='?' (Size = 256)"], CommandType='Text', CommandTimeout='30']"
""SELECT TOP(1) [u].[Id], [u].[AccessFailedCount], [u].[ConcurrencyStamp], [u].[Configuration], [u].[CreatedBy], [u].[CreatedDate], [u].[Email], [u].[EmailConfirmed], [u].[FullName], [u].[IsEnabled], [u].[JobTitle], [u].[LockoutEnabled], [u].[LockoutEnd], [u].[NormalizedEmail], [u].[NormalizedUserName], [u].[PasswordHash], [u].[PhoneNumber], [u].[PhoneNumberConfirmed], [u].[SecurityStamp], [u].[TwoFactorEnabled], [u].[UpdatedBy], [u].[UpdatedDate], [u].[UserName]
FROM [AspNetUsers] AS [u]
WHERE [u].[NormalizedUserName] = @__normalizedUserName_0" (0723d8ff)
2020-05-18T23:08:37.2421364+07:00 0HLVR8MAKOAMO:00000003 [INF] Executed DbCommand ("3"ms) [Parameters=["@__user_Id_0='?' (Size = 450)"], CommandType='Text', CommandTimeout='30']"
""SELECT [uc].[Id], [uc].[ClaimType], [uc].[ClaimValue], [uc].[UserId]
FROM [AspNetUserClaims] AS [uc]
WHERE [uc].[UserId] = @__user_Id_0" (0723d8ff)
2020-05-18T23:08:37.3202716+07:00 0HLVR8MAKOAMO:00000003 [INF] Executed DbCommand ("4"ms) [Parameters=["@__userId_0='?' (Size = 450)"], CommandType='Text', CommandTimeout='30']"
""SELECT [role].[Name]
FROM [AspNetUserRoles] AS [userRole]
INNER JOIN [AspNetRoles] AS [role] ON [userRole].[RoleId] = [role].[Id]
WHERE [userRole].[UserId] = @__userId_0" (0723d8ff)
2020-05-18T23:08:37.3388618+07:00 0HLVR8MAKOAMO:00000003 [INF] Executed DbCommand ("1"ms) [Parameters=["@__normalizedName_0='?' (Size = 256)"], CommandType='Text', CommandTimeout='30']"
""SELECT TOP(1) [r].[Id], [r].[ConcurrencyStamp], [r].[CreatedBy], [r].[CreatedDate], [r].[Description], [r].[Name], [r].[NormalizedName], [r].[UpdatedBy], [r].[UpdatedDate]
FROM [AspNetRoles] AS [r]
WHERE [r].[NormalizedName] = @__normalizedName_0" (0723d8ff)
2020-05-18T23:08:37.3672173+07:00 0HLVR8MAKOAMO:00000003 [INF] Executed DbCommand ("2"ms) [Parameters=["@__role_Id_0='?' (Size = 450)"], CommandType='Text', CommandTimeout='30']"
""SELECT [rc].[ClaimType], [rc].[ClaimValue]
FROM [AspNetRoleClaims] AS [rc]
WHERE [rc].[RoleId] = @__role_Id_0" (0723d8ff)
2020-05-18T23:08:37.3952458+07:00 0HLVR8MAKOAMO:00000003 [INF] Executing SignInResult with authentication scheme ("ASOS") and the following principal: "System.Security.Claims.ClaimsPrincipal". (f8bf0a82)
2020-05-18T23:08:37.6749200+07:00 0HLVR8MAKOAMO:00000003 [INF] Executed DbCommand ("6"ms) [Parameters=["@p0='?' (Size = 450), @p1='?' (Size = 450), @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (Size = 4000), @p5='?' (Size = 4000), @p6='?' (Size = 4000)"], CommandType='Text', CommandTimeout='30']"
""SET NOCOUNT ON;
INSERT INTO [OpenIddictAuthorizations] ([Id], [ApplicationId], [ConcurrencyToken], [Scopes], [Status], [Subject], [Type])
VALUES (@p0, @p1, @p2, @p3, @p4, @p5, @p6);" (0723d8ff)
2020-05-18T23:08:37.7137252+07:00 0HLVR8MAKOAMO:00000003 [INF] An ad hoc authorization was automatically created and associated with an unknown application: "e4e038fd-a5aa-405a-ab30-a697796e8d69". (b5fe4ad7)
2020-05-18T23:08:37.8731077+07:00 0HLVR8MAKOAMO:00000003 [INF] Executed DbCommand ("17"ms) [Parameters=["@p0='?' (Size = 450), @p1='?' (Size = 450), @p2='?' (Size = 450), @p3='?' (Size = 4000), @p4='?', @p5='?', @p6='?' (Size = 4000), @p7='?' (Size = 450), @p8='?' (Size = 4000), @p9='?' (Size = 4000), @p10='?' (Size = 4000)"], CommandType='Text', CommandTimeout='30']"
""SET NOCOUNT ON;
INSERT INTO [OpenIddictTokens] ([Id], [ApplicationId], [AuthorizationId], [ConcurrencyToken], [CreationDate], [ExpirationDate], [Payload], [ReferenceId], [Status], [Subject], [Type])
VALUES (@p0, @p1, @p2, @p3, @p4, @p5, @p6, @p7, @p8, @p9, @p10);" (0723d8ff)
2020-05-18T23:08:37.9207036+07:00 0HLVR8MAKOAMO:00000003 [INF] The token response was successfully returned: "{
  \"token_type\": \"Bearer\",
  \"access_token\": \"[removed for security reasons]\",
  \"expires_in\": 3600,
  \"refresh_token\": \"[removed for security reasons]\",
  \"id_token\": \"[removed for security reasons]\"
}". (12d98310)
2020-05-18T23:08:37.9548969+07:00 0HLVR8MAKOAMO:00000003 [INF] Executed action "QLNS.Controllers.AuthorizationController.Exchange (QLNS)" in 1915.7465ms (afa2e885)
2020-05-18T23:08:37.9813714+07:00 0HLVR8MAKOAMO:00000003 [INF] Request finished in 2164.7367ms 200 application/json;charset=UTF-8 (791a596a)
2020-05-18T23:08:38.0067257+07:00 0HLVR8MAKOAMO:00000004 [INF] Request starting HTTP/1.1 GET http://localhost:55842/api/DonVis/getDonViByRole/0280636e-99e3-4b3d-bec0-5e9aa67ad2b5 application/json  (ca22a1cb)
2020-05-18T23:08:38.0586905+07:00 0HLVR8MAKOAMP:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:55842/images/icon/user-image.png   (ca22a1cb)
2020-05-18T23:08:38.0618055+07:00 0HLVR8MAKOAMP:00000001 [INF] Sending file. Request path: '"/images/icon/user-image.png"'. Physical path: '"D:\PMQLHienVat\source\QLNS\QLNS\wwwroot\images\icon\user-image.png"' (27b0a520)
2020-05-18T23:08:38.0668653+07:00 0HLVR8MAKOAMQ:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:55842/images/banner.png   (ca22a1cb)
2020-05-18T23:08:38.0670334+07:00 0HLVR8MAKOAMQ:00000001 [INF] Sending file. Request path: '"/images/banner.png"'. Physical path: '"D:\PMQLHienVat\source\QLNS\QLNS\wwwroot\images\banner.png"' (27b0a520)
2020-05-18T23:08:38.0685690+07:00 0HLVR8MAKOAMP:00000001 [INF] Request finished in 9.6472ms 200 image/png (791a596a)
2020-05-18T23:08:38.0692860+07:00 0HLVR8MAKOAMQ:00000001 [INF] Request finished in 2.4202ms 200 image/png (791a596a)
2020-05-18T23:08:38.0837487+07:00 0HLVR8MAKOAMO:00000004 [INF] AuthenticationScheme: "Bearer" was successfully authenticated. (1805f3b3)
2020-05-18T23:08:38.1511421+07:00 0HLVR8MAKOAMO:00000004 [INF] Executing action method "QLNS.Controllers.DonVisController.GetDonViByRole (QLNS)" with arguments (["0280636e-99e3-4b3d-bec0-5e9aa67ad2b5"]) - ModelState is Valid (ba7f4ac2)
2020-05-18T23:08:38.1541060+07:00 0HLVR8MAKOAMO:00000004 [INF] Entity Framework Core "2.0.1-rtm-125" initialized '"ApplicationDbContext"' using provider '"Microsoft.EntityFrameworkCore.SqlServer"' with options: "MigrationsAssembly=UrencoDB " (9958f5bb)
2020-05-18T23:08:38.2511427+07:00 0HLVR8MAKOAMO:00000004 [INF] Executed DbCommand ("37"ms) [Parameters=["@p0='?' (Size = 4000), @p1='?'"], CommandType='Text', CommandTimeout='30']"
""GetDonViByRole @p0,@p1" (0723d8ff)
2020-05-18T23:08:38.2723676+07:00 0HLVR8MAKOAMO:00000004 [ERR] An exception occurred in the database while iterating the results of a query for context type '"DAL.ApplicationDbContext"'."
""System.InvalidOperationException: The required column 'LoaiDonVi' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Sql.Internal.FromSqlNonComposedQuerySqlGenerator.CreateValueBufferFactory(IRelationalValueBufferFactoryFactory relationalValueBufferFactoryFactory, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Internal.NonCapturingLazyInitializer.EnsureInitialized[TParam,TValue](TValue& target, TParam param, Func`2 valueFactory)
   at Microsoft.EntityFrameworkCore.Query.Internal.ShaperCommandContext.NotifyReaderCreated(DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.BufferlessMoveNext(Boolean buffer)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.MoveNext()
   at Microsoft.EntityFrameworkCore.Query.Internal.LinqOperatorProvider._TrackEntities[TOut,TIn](IEnumerable`1 results, QueryContext queryContext, IList`1 entityTrackingInfos, IList`1 entityAccessors)+MoveNext()
   at Microsoft.EntityFrameworkCore.Query.Internal.LinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.MoveNext()" (64e2593c)
System.InvalidOperationException: The required column 'LoaiDonVi' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Sql.Internal.FromSqlNonComposedQuerySqlGenerator.CreateValueBufferFactory(IRelationalValueBufferFactoryFactory relationalValueBufferFactoryFactory, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Internal.NonCapturingLazyInitializer.EnsureInitialized[TParam,TValue](TValue& target, TParam param, Func`2 valueFactory)
   at Microsoft.EntityFrameworkCore.Query.Internal.ShaperCommandContext.NotifyReaderCreated(DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.BufferlessMoveNext(Boolean buffer)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.MoveNext()
   at Microsoft.EntityFrameworkCore.Query.Internal.LinqOperatorProvider._TrackEntities[TOut,TIn](IEnumerable`1 results, QueryContext queryContext, IList`1 entityTrackingInfos, IList`1 entityAccessors)+MoveNext()
   at Microsoft.EntityFrameworkCore.Query.Internal.LinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.MoveNext()
2020-05-18T23:08:38.2749971+07:00 0HLVR8MAKOAMO:00000004 [INF] Executed action "QLNS.Controllers.DonVisController.GetDonViByRole (QLNS)" in 177.9377ms (afa2e885)
2020-05-18T23:08:38.2891363+07:00 0HLVR8MAKOAMO:00000004 [ERR] An unhandled exception has occurred: The required column 'LoaiDonVi' was not present in the results of a 'FromSql' operation. (e70bd093)
System.InvalidOperationException: The required column 'LoaiDonVi' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Sql.Internal.FromSqlNonComposedQuerySqlGenerator.CreateValueBufferFactory(IRelationalValueBufferFactoryFactory relationalValueBufferFactoryFactory, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Internal.NonCapturingLazyInitializer.EnsureInitialized[TParam,TValue](TValue& target, TParam param, Func`2 valueFactory)
   at Microsoft.EntityFrameworkCore.Query.Internal.ShaperCommandContext.NotifyReaderCreated(DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.BufferlessMoveNext(Boolean buffer)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.MoveNext()
   at Microsoft.EntityFrameworkCore.Query.Internal.LinqOperatorProvider._TrackEntities[TOut,TIn](IEnumerable`1 results, QueryContext queryContext, IList`1 entityTrackingInfos, IList`1 entityAccessors)+MoveNext()
   at Microsoft.EntityFrameworkCore.Query.Internal.LinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.MoveNext()
   at System.Collections.Generic.List`1.AddEnumerable(IEnumerable`1 enumerable)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at QLNS.Controllers.DonVisController.GetDonViByRole(String userId) in D:\PMQLHienVat\source\QLNS\QLNS\Controllers\DonViController.cs:line 30
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIIndexMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.Invoke(HttpContext context)
2020-05-18T23:08:38.3024235+07:00 0HLVR8MAKOAMO:00000004 [INF] Request finished in 295.6351ms 500 application/json (791a596a)
2020-05-18T23:08:38.4675655+07:00 0HLVR8MAKOAMQ:00000002 [INF] Request starting HTTP/1.1 POST http://localhost:55842/api/Logs application/json 126 (ca22a1cb)
2020-05-18T23:08:38.4682958+07:00 0HLVR8MAKOAMQ:00000002 [INF] Policy execution successful. (0204925d)
2020-05-18T23:08:38.4707435+07:00 0HLVR8MAKOAMQ:00000002 [INF] AuthenticationScheme: "Bearer" was successfully authenticated. (1805f3b3)
2020-05-18T23:08:38.6060270+07:00 0HLVR8MAKOAMQ:00000002 [INF] Executing action method "QLNS.Controllers.LogsController.PostLog (QLNS)" with arguments (["DAL.Models.Log"]) - ModelState is Valid (ba7f4ac2)
2020-05-18T23:08:38.6087448+07:00 0HLVR8MAKOAMQ:00000002 [INF] Entity Framework Core "2.0.1-rtm-125" initialized '"ApplicationDbContext"' using provider '"Microsoft.EntityFrameworkCore.SqlServer"' with options: "MigrationsAssembly=UrencoDB " (9958f5bb)
2020-05-18T23:08:38.6489567+07:00 0HLVR8MAKOAMQ:00000002 [INF] Executed DbCommand ("5"ms) [Parameters=["@p0='?' (Size = 4000), @p1='?', @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (Size = 4000)"], CommandType='Text', CommandTimeout='30']"
""SET NOCOUNT ON;
INSERT INTO [tbl_Log] ([HanhDong], [NgayThucHien], [NguoiThucHien], [NoiDung], [TenModule])
VALUES (@p0, @p1, @p2, @p3, @p4);
SELECT [LogID]
FROM [tbl_Log]
WHERE @@ROWCOUNT = 1 AND [LogID] = scope_identity();" (0723d8ff)
2020-05-18T23:08:38.6899562+07:00 0HLVR8MAKOAMQ:00000002 [INF] Executing ObjectResult, writing value "Microsoft.AspNetCore.Mvc.ControllerContext". (4e968210)
2020-05-18T23:08:38.7365963+07:00 0HLVR8MAKOAMQ:00000002 [INF] Executed action "QLNS.Controllers.LogsController.PostLog (QLNS)" in 231.1505ms (afa2e885)
2020-05-18T23:08:38.7415429+07:00 0HLVR8MAKOAMQ:00000002 [INF] Request finished in 274.3328ms 201 application/json; charset=utf-8 (791a596a)
2020-05-18T23:17:31.4505264+07:00  [INF] webpack building... (478286db)
2020-05-18T23:17:35.9528679+07:00  [INF] webpack built 1aa7c2bca7a7ec26987f in 4299ms (8e3f559f)
2020-05-18T23:17:36.3158671+07:00  [INF] webpack building... (478286db)
2020-05-18T23:17:37.8434287+07:00  [INF] webpack built 1aa7c2bca7a7ec26987f in 1277ms (07cad863)
